### PR TITLE
Update docs for import_vcf about filters and pass

### LINKED ISSUE #1657: fix docs re: va.pass

### PR DESCRIPTION
**Pull request by tpoterba**
Fixes #1657

### CODE REVIEW

File: python/hail/context.py [hunk 1]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
+            >>> pass_vds = vds.filter_variants_expr('va.filters.isEmpty()', keep=True)
+
         **Annotations**
 
-        - **va.pass** (*Boolean*) -- true if the variant contains `PASS` in the filter field (false if `.` or other)
-        - **va.filters** (*Set[String]*) -- set containing the list of filters applied to a variant. Accessible using `va.filters.contains("VQSRTranche99.5...")`, for example
-        - **va.rsid** (*String*) -- rsid of the variant, if it has one ("." otherwise)
-        - **va.qual** (*Double*) -- the number in the qual field
-        - **va.info** (*T*) -- matches (with proper capitalization) any defined info field. Data types match the type specified in the vcf header, and if the `Number` is "A", "R", or "G", the result will be stored in an array (accessed with array\[index\]).
+        - **va.filters** (*Set[String]*) -- The containing the list of filters applied to a variant.
```
Comment (johnc1231) [inline python/hail/context.py]: Missing a word here. "The containing".

File: python/hail/context.py [hunk 2]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
+          struct ``va.info``. Data types match the type specified in the vcf header, and if 
+          the ``Number`` is not 1, the result will be stored as an array. 
+
 
         :param path: VCF file(s) to read.
         :type path: str or list of str
 
-        :param bool force: If True, load .gz files serially.
+        :param bool force: If True, load .gz files serially. This means that no downstream operations
+            will parallelized, so this is not recommended for a VCF larger than a few MB.
```
Comment (johnc1231) [inline python/hail/context.py]: "will be parallelized"

File: python/hail/context.py [hunk 3]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
+            >>> pass_vds = vds.filter_variants_expr('va.filters.isEmpty()', keep=True)
+
         **Annotations**
 
-        - **va.pass** (*Boolean*) -- true if the variant contains `PASS` in the filter field (false if `.` or other)
-        - **va.filters** (*Set[String]*) -- set containing the list of filters applied to a variant. Accessible using `va.filters.contains("VQSRTranche99.5...")`, for example
-        - **va.rsid** (*String*) -- rsid of the variant, if it has one ("." otherwise)
-        - **va.qual** (*Double*) -- the number in the qual field
-        - **va.info** (*T*) -- matches (with proper capitalization) any defined info field. Data types match the type specified in the vcf header, and if the `Number` is "A", "R", or "G", the result will be stored in an array (accessed with array\[index\]).
+        - **va.filters** (*Set[String]*) -- The set containing all filters applied to a variant.
```
Comment (jbloom22) [inline python/hail/context.py]: Set containing

File: python/hail/context.py [hunk 4]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
+
         **Annotations**
 
-        - **va.pass** (*Boolean*) -- true if the variant contains `PASS` in the filter field (false if `.` or other)
-        - **va.filters** (*Set[String]*) -- set containing the list of filters applied to a variant. Accessible using `va.filters.contains("VQSRTranche99.5...")`, for example
-        - **va.rsid** (*String*) -- rsid of the variant, if it has one ("." otherwise)
-        - **va.qual** (*Double*) -- the number in the qual field
-        - **va.info** (*T*) -- matches (with proper capitalization) any defined info field. Data types match the type specified in the vcf header, and if the `Number` is "A", "R", or "G", the result will be stored in an array (accessed with array\[index\]).
+        - **va.filters** (*Set[String]*) -- The set containing all filters applied to a variant. 
+        - **va.rsid** (*String*) -- The rsID of the variant, if it has one ("." otherwise).
```
Comment (jbloom22) [inline python/hail/context.py]: rsId of

File: python/hail/context.py [hunk 5]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
+        - **va.qual** (*Double*) -- The floating-point number in the QUAL field.
```
Comment (jbloom22) [inline python/hail/context.py]: Floating-point

File: python/hail/context.py (hunk 4 above)
Comment (jbloom22) [inline python/hail/context.py]: '.'

File: python/hail/context.py [hunk 6]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
             Imported as: 2/2:0,0,9:9:24:24,1000,40,1000:1000:0
 
 
+        .. note::
+            
+            Using the **FILTER** field:
+            
+            The information in the FILTER field of a VCF is contained in the ``va.filters`` annotation.
+            This annotation is a ``Set``, and can be queried for filter membership with expressions
```
Comment (jbloom22) [inline python/hail/context.py]: remove ,

File: python/hail/context.py [hunk 7]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
             Imported as: 2/2:0,0,9:9:24:24,1000,40,1000:1000:0
 
 
+        .. note::
+            
+            Using the **FILTER** field:
+            
+            The information in the FILTER field of a VCF is contained in the ``va.filters`` annotation.
+            This annotation is a ``Set``, and can be queried for filter membership with expressions 
+            like: ``va.filters.contains("VQSRTranche99.5...")``. A variant that was flagged as "PASS"
```
Comment (jbloom22) [inline python/hail/context.py]: remove :

File: python/hail/context.py [hunk 8]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
 
 
+        .. note::
+            
+            Using the **FILTER** field:
+            
+            The information in the FILTER field of a VCF is contained in the ``va.filters`` annotation.
+            This annotation is a ``Set``, and can be queried for filter membership with expressions 
+            like: ``va.filters.contains("VQSRTranche99.5...")``. A variant that was flagged as "PASS" 
+            will have no filters applied: for these variants, `va.filters.isEmpty()` is true. Thus,
```
Comment (jbloom22) [inline python/hail/context.py]: change : to ;

Comment (jbloom22) [inline python/hail/context.py]: double back-ticks

File: python/hail/context.py [hunk 9]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
+            Using the **FILTER** field:
+            
+            The information in the FILTER field of a VCF is contained in the ``va.filters`` annotation.
+            This annotation is a ``Set``, and can be queried for filter membership with expressions 
+            like: ``va.filters.contains("VQSRTranche99.5...")``. A variant that was flagged as "PASS" 
+            will have no filters applied: for these variants, `va.filters.isEmpty()` is true. Thus, 
+            filtering to PASS variants can be done with :py:meth:`.VariantDataset.filter_variants_expr`
+            as follows:
+            
+            >>> pass_vds = vds.filter_variants_expr('va.filters.isEmpty()', keep=True)
```
Comment (jbloom22) [inline python/hail/context.py]: you don't need keep=True

File: python/hail/context.py [hunk 10]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
-        - **va.pass** (*Boolean*) -- true if the variant contains `PASS` in the filter field (false if `.` or other)
-        - **va.filters** (*Set[String]*) -- set containing the list of filters applied to a variant. Accessible using `va.filters.contains("VQSRTranche99.5...")`, for example
-        - **va.rsid** (*String*) -- rsid of the variant, if it has one ("." otherwise)
-        - **va.qual** (*Double*) -- the number in the qual field
-        - **va.info** (*T*) -- matches (with proper capitalization) any defined info field. Data types match the type specified in the vcf header, and if the `Number` is "A", "R", or "G", the result will be stored in an array (accessed with array\[index\]).
+        - **va.filters** (*Set[String]*) -- The set containing all filters applied to a variant. 
+        - **va.rsid** (*String*) -- The rsID of the variant, if it has one ("." otherwise).
+        - **va.qual** (*Double*) -- The floating-point number in the QUAL field.
+        - **va.info** (*Struct*) -- All INFO fields defined in the VCF header can be found in the
+          struct ``va.info``. Data types match the type specified in the vcf header, and if
```
Comment (jbloom22) [inline python/hail/context.py]: capital VCF

File: python/hail/context.py [hunk 11]
```diff
@@ -459,18 +459,34 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
-        - **va.filters** (*Set[String]*) -- set containing the list of filters applied to a variant. Accessible using `va.filters.contains("VQSRTranche99.5...")`, for example
-        - **va.rsid** (*String*) -- rsid of the variant, if it has one ("." otherwise)
-        - **va.qual** (*Double*) -- the number in the qual field
-        - **va.info** (*T*) -- matches (with proper capitalization) any defined info field. Data types match the type specified in the vcf header, and if the `Number` is "A", "R", or "G", the result will be stored in an array (accessed with array\[index\]).
+        - **va.filters** (*Set[String]*) -- The set containing all filters applied to a variant. 
+        - **va.rsid** (*String*) -- The rsID of the variant, if it has one ("." otherwise).
+        - **va.qual** (*Double*) -- The floating-point number in the QUAL field.
+        - **va.info** (*Struct*) -- All INFO fields defined in the VCF header can be found in the
+          struct ``va.info``. Data types match the type specified in the vcf header, and if 
+          the ``Number`` is not 1, the result will be stored as an array.
```
Comment (jbloom22) [inline python/hail/context.py]: don't need the in from of ``Number``

File: python/hail/context.py (hunk 4 above)
Comment (tpoterba) [inline python/hail/context.py]: rsID is convention

File: python/hail/context.py [hunk 12]
```diff
@@ -464,22 +464,22 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
             Using the **FILTER** field:
             
             The information in the FILTER field of a VCF is contained in the ``va.filters`` annotation.
-            This annotation is a ``Set``, and can be queried for filter membership with expressions 
-            like: ``va.filters.contains("VQSRTranche99.5...")``. A variant that was flagged as "PASS" 
-            will have no filters applied: for these variants, `va.filters.isEmpty()` is true. Thus, 
+            This annotation is a ``Set`` and can be queried for filter membership with expressions 
+            like ``va.filters.contains("VQSRTranche99.5...")``. Variant that are flagged as "PASS"
```
Comment (jbloom22) [inline python/hail/context.py]: Variants

**Pull request merged**